packages/fx-core/src/plugins/solution/fx-solution/commonQuestions.ts

```diff
@@ -142,7 +142,7 @@ export async function checkM365Tenant(
     return err(
       new UserError(
         SolutionError.TeamsAppTenantIdNotRight,
-        `The m365 tenant id '${m365TenantId}' is not found in the current account.`,
+        `The M365 tenant id '${m365TenantId}'(which is used before) does not match the current account, please use the right M365 account.`,
```

Comment:
How about including env name in this error message? So user can know the M365 tenant id in which env doesn't matched.

Reply:
Already add.